node/silkworm/downloader/block_exchange.cpp

```diff
@@ -121,4 +125,12 @@ void BlockExchange::log_status() {
                  << "; stats: " << body_sequence_.statistics();
 }
 
+void BlockExchange::send_penalization(PeerId id, Penalty p) noexcept {
+    using namespace std::chrono_literals;
+    rpc::PenalizePeer penalize_peer(id, p);
+    penalize_peer.do_not_throw_on_failure();
+    penalize_peer.timeout(1s);
```

Comment:
A constant in `block_exchange.hpp` instead of a magic number in code would be preferable here.

Reply:
done